main.go

```diff
@@ -35,16 +41,61 @@ func main() {
 	fmt.Println()
 	log.Infof("Running test")
 
-	testCmd := command.New("flutter", append([]string{"test"}, additionalParams...)...).
+	pr, pw := io.Pipe()
+
+	if _, err := exec.LookPath("tojunit"); err != nil {
+		log.Infof("Command `tojunit` not found, installing...")
+		junitInstallCmd := command.New("flutter", append([]string{"pub", "global", "activate", "junitreport"})...).
+			SetStdout(os.Stdout).
+			SetStderr(os.Stderr).
+			SetDir(cfg.ProjectLocation)
+		if err := junitInstallCmd.Run(); err != nil {
+			failf("Command `tojunit` failed to install, error: %s", err)
```

Comment:
It is reccomended to check the type of error (process exited with 1, or failed to start process). If not printing to Stdout, this can help prevent logging an empty output also.
You can omit the `, error:` part.

For example:
```
if err = junitInstallCmd.Run(); err != nil {
		if errorutil.IsExitStatusError(err) {
			failf("Install command failed, %s", err)
		}
		failf("Failed to run command, %s", err)
	}
```


https://github.com/bitrise-steplib/steps-yarn/blob/c5de117c0d4fa3f306e6e1e0fea8dd5d4794e011/main.go#L153

Reply:
done